package dk.aau.astep.appserver.dataaccess.api.exceptions;

import dk.aau.astep.appserver.dataaccess.api.exceptions.DatabaseException;
import org.junit.Test;

public class ExceptionTest {
  // The build will fail when tests are forced to fail!
  /*
  @Test(expected = DatabaseException.class)
  public void DatabaseExceptionTest(){
    try {
      //mustThrowException();
      //fail();
    } catch (ApiExceptionHandler e) {
      // expected
      // could also check for message of exception, etc.
    }
  }
  */
}
